scheduled-jobs/build/ose-master-full/Jenkinsfile

```diff
@@ -0,0 +1,19 @@
+
+properties([
+    buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '30', numToKeepStr: '')),
+    disableConcurrentBuilds(),
+])
+
+currentBuild.displayName += "4.1 full build"
+def b = build(
+    job: '../aos-cd-builds/build%2Fose4.1',
```

Comment:
Just talking out loud here - We should put some data in `ocp-build-data:master` that specifies what version master is and things like that. Single yaml file that jobs like this could curl and read. That way version are no longer hard coded in jobs and we don't have to update them anymore when version bumps happen.

Reply:
We'd still have to create e.g. the `ose4.2` job and repoint others when that time rolls around. I don't see updating this job at the same time as a problem worth the trouble of automating away.

The idea with `ocp-build-data:master` seems good though. It would be good for example to be able to have the commonlib options for which versions are available key off... which versions are available.

Reply:
Well, to explain my full vision, eventually I would like "One Scheduled Job to Rule Them All" that just runs every ~15 min, looks at `ocp-build-data:master` and then fires off whatever jobs are required.

Reply:
Sounds good :)